usbreset.c

```diff
@@ -89,6 +90,10 @@ static struct usbentry *parse_devlist(DIR *d)
 	if (attr)
 		strncpy(dev.product_name, attr, sizeof(dev.product_name) - 1);
 
+	attr = sysfs_attr(e->d_name, "serial");
+	if (attr)
+		snprintf(dev.serial, sizeof(dev.serial) - 1, "SN:%s", attr);
```

Comment:
Try not to add "SN:" to the serial number itself, as it's not done that way for the other attributes (name, id, etc.)  It can be used for the command line, but not for carrying around within the code.

Reply:
I changed it in a573189.